tools/geojson_simplifier/download.py

```diff
@@ -0,0 +1,120 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Downloads and saves GeoJson map files from DataCommons.
+    Typical usage:
+    python3 download.py
+"""
+
+import datacommons as dc
+import geojson
+
+
+# TODO(fpernice-google): Support downloading more than just US states.
+class GeojsonDownloader:
+    """Downloads desired GeoJSON files from the DataCommons Knowledge Graph.
+
+    Attributes:
+        geojsons: A dictionary that maps each queried area to another
+                  dictionary containing the GeoJSON coordinate information. An
+                  example of this is the following
+
+                {   # Multipolygon of the state of Alabama (fake).
+                    "geoId/01": [{
+                        "type": "MultiPolygon",
+
+                        # Set of individual Polygons that compose it.
+                        "coordinates": [
+                            # Polygon 1
+                            [[ [1.5, 12.4], [5.3, 45.2], [1.1, 3.5],
+                                                            [1.5, 12.4] ]],
+                            # Polygon 2
+                            [[ [1, 2], [3, 4], [5, 6], [2, -1], [1, 2] ]],
+                            # Polygon 3
+                            [[ [53, 23], [65, 2], [31, 12], [53, 23] ]]
+                        ]
+                    }],
+                    # Polygon of the state of Wyoming (fake).
+                    # Since Wyoming is a single chunk of land, its type
+                    # is Polygon instead of Multipolygon.
+                    "geoId/17": [{
+                        "type": "Polygon",
+                        "coordinates": [
+                            # Polygon 1
+                            [[ [1.5, 12.4], [5.3, 45.2], [1.1, 3.5],
+                                                            [1.5, 12.4] ]]
+                        ]
+                    }]
+
+                }
+    """
+    LEVEL_MAP = {
+        "Country": "AdministrativeArea1",
+        "AdministrativeArea1": "AdministrativeArea2",
+        "AdministrativeArea2": "City"
+    }
+
+    def __init__(self):
+        dc.set_api_key('dev')
+        self.geojsons = None
+
+    def download_data(self, place='country/USA'):
+        """Downloads GeoJSON data for a specified location.
+
+        Given the specified location, extracts the GeoJSONs of all
+        administrative areas one level below it (as specified by the
+        LEVEL_MAP class constant). For example, if the input is country/USA,
+        extracts all AdministrativeArea1's within the US (US states).
+
+        Args:
+            place: A string that is a valid value for the geoId property of a
+                   DataCommons node.
+
+        Raises:
+            ValueError: If a Data Commons API call fails.
+        """
+        geolevel_below = dc.get_property_values([place],
```

Comment:
Is there a better name you can use to describe this variable? It's really the geo level of the specified place, not below..

Reply:
Good catch! I had copied some of this code from Ian's code and I forgot to change the variable name :/